dev/TEAL.md

```diff
@@ -49,6 +58,30 @@ Constants are loaded into the environment by two opcodes, `intcblock` and `bytec
 
 Constants are pushed onto the stack by `intc`, `intc_[0123]`, `bytec`, and `bytec_[0123]`. The assembler will handle converting `int N` or `byte N` into the appropriate form of the instruction needed.
 
+### Named Integer Constants
+
+#### OnComplete
+| Value | Constant name | Description |
+| --- | --- | --- |
+| 0 | NoOp | Application transaction will simply call its ApprovalProgram. |
```

Comment:
Confusing, change to: "Application will *only* call its ApprovalProgram, the application state will change as prescribed by this program but no other action will be taken".

Reply:
I'll change this to be consistent with the text from `ledger.md`, as you suggested below.